python/py-torchaudio/Portfile

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        pytorch audio 0.8.1 v
+revision            0
+name                py-[strsed ${github.author} {s|^py||}]${github.project}
```

Comment:
please just set this directly to `py-torchaudio`; that's immediately clear for everyone and there is no need to be overly clever here

Reply:
Yeah, I also noticed this, and almost made the same comment in other ports that did the same, but in the end let it pass. Normally I would be the first to support removing code/variable duplication, but here yeah its probably better to just write it out in plain text.

Reply:
*sigh* one less BSD regex in the world.